.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Release
+on:
+  release:
+    types: [published]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: v1/${{ runner.os }}/pip/${{ hashFiles('{requirements,development}.txt') }}
+          restore-keys: v1/${{ runner.os }}/pip/
+      - run: make release
+        env:
+          PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
```

Comment:
Let's make sure this secret is set before we cut the next release

Reply:
Maybe even before this PR is merged

Reply:
Done